src/bun.js/ipc.zig

```diff
@@ -724,18 +888,19 @@ fn NewNamedPipeIPCHandler(comptime Context: type) type {
                     return;
                 },
                 .result => {
+                    client.setBlocking(false);
```

Comment:
Was this intended to still be here? Or is this from before? cc @cirospaciari

Reply:
This is from before we dont need to setBlocking anymore

Reply:
roger